invenio_github/utils.py

```diff
@@ -114,21 +65,27 @@ def get_contributors(gh, repo_id):
         if contributors_iter.last_status == 200:
 
             def get_author(contributor):
-                r = requests.get(contributor['url'])
+                r = requests.get(contributor["url"])
                 if r.status_code == 200:
                     data = r.json()
                     return dict(
-                        name=(data['name'] if 'name' in data and data['name']
-                              else data['login']),
-                        affiliation=data.get('company') or '',
+                        name=(
+                            data["name"]
+                            if "name" in data and data["name"]
+                            else data["login"]
+                        ),
+                        affiliation=data.get("company") or "",
                     )
+
             # Sort according to number of contributions
             contributors = sorted(
-                contributors,
-                key=lambda x: x.as_dict()['contributions'],
-                reverse=True)
-            contributors = [get_author(x.as_dict()) for x in contributors[:30]
-                            if x.as_dict()['type'] == 'User']
+                contributors, key=lambda x: x.as_dict()["contributions"], reverse=True
+            )
+            contributors = [
+                get_author(x.as_dict())
+                for x in contributors[:30]
```

Comment:
why is there a 30 limit? should it be configurable?

Reply:
Added a config 👍 

Why 30? I am not sure